hack/replace.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+read -a modules <<< $(go list -m)
+read -a module_paths <<< $(go list -m -f {{.Dir}})
```

Comment:
I'd like to move this into `Makefile` to avoid re-calculating it for each module, but I don't know how to export variables in `Makefile` so that they're accessible here.